spec/nameable_spec.rb

```diff
@@ -1,9 +1,13 @@
-context("convenience") do
-  it 'to_s' do
-    expect(Nameable("Chris Horn")).to be_a(String)
-  end
+require 'spec_helper'
+
+describe Nameable do
+  context("convenience methods") do
+    it 'to_s' do
+      expect(Nameable("Chris Horn")).to be_a(String)
+    end
 
-  it 'parse' do
-    expect(Nameable.parse("Chris Horn")).to be_a(Nameable::Latin)
+    it 'parse' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.